AdaptorsDataAccessPlugins/manifest.mf

```diff
@@ -0,0 +1,6 @@
+Manifest-Version: 1.0
```

Comment:
Unless you require the ability to turn this functionality off specifically, i'd suggest moving all this code under `AdaptorsFunctionality/src/au/gov/asd/tac/constellation/functionality/adaptors/dataaccess/plugins/`.